.ebextensions/configure-nginx-token-auth.config

```diff
@@ -1,8 +1,8 @@
 commands:
-  01-get-token-auth_file:
-    command: "aws --region=us-west-2 s3 cp s3://solr.ucldc/token_auth /var/app/"
-  02-move-to-nginx-dir:
-    command: "mv /var/app/token_auth /etc/nginx/"
+#  01-get-token-auth_file:
+#    command: "aws --region=us-west-2 s3 cp s3://solr.ucldc/token_auth /var/app/"
+#  02-move-to-nginx-dir:
+#    command: "mv /var/app/token_auth /etc/nginx/"
```

Comment:
could just remove these lines

Reply:
okay. I'm thinking of moving to .platform as well.